Added troubleshooting guide including cm-cmk support (#11)
aws-samples/emr-serverless-samples: utilities/spark-ui/README.md

## Start the Spark History Server

> You can use a pair of AWS access key and secret key, or temporary AWS credentials. This credentials should have access to s3 bucket. If customer enables encryption for the logs stored in s3 bucket, then credentials should have access to KMS key as well.

Reviewer: ```suggestion
You can use a pair of AWS access key and secret key, or temporary AWS credentials. These credentials should have access to the S3 log bucket. If encryption is enabled for the logs stored in the S3 bucket, these credentials should have access to the necessary KMS key as well.
```

    emr/spark-ui
```

4. Access the Spark UI via http://localhost:18080

## Troubleshooting

You may get following exception during SHS startup.

1. **Issue/Exception:** com.amazon.ws.emr.hadoop.fs.shaded.com.amazonaws.services.s3.model.AmazonS3Exception: The ciphertext refers to a customer master key that does not exist, does not exist in this region, or you are not allowed to access. (Service: Amazon S3; Status Code: 403; Error Code: AccessDenied) 
   

>    **Reason:** Given user credentials may not have the access to KMS key which used to encrypt the logs in s3 bucket. Add kms policy with decrypt permission and verify.

Reviewer: ```suggestion
   **Reason:** Given user credentials may not have the access to KMS key which is used to encrypt the logs in the S3 bucket. Add KMS policy with decrypt permission and verify.
```

2. **Issue/Exception:**  com.amazon.ws.emr.hadoop.fs.shaded.com.amazonaws.services.s3.model.AmazonS3Exception: Access Denied (Service: Amazon S3; Status Code: 403; Error Code: AccessDenied; Request ID: 9VRVX4FX3ETVQ47T; S3 Extended Request ID: pxyErt0+HhMfrBye5fokQe1H6TynqIpSHM6YNdXg87DZA4Tji5kl6xh3leYpNb2Ej+plDXVJQsY=; Proxy: null) 

>    **Reason:** Given user credentials may not have the access s3 bucket. Add s3 policy with read permission and verify.

Reviewer: ```suggestion
   **Reason:** Given user credentials may not have the access the S3 bucket. Add S3 policy with read permission and verify.
```
